pymobiledevice3/services/web_protocol/inspector_session.py

```diff
@@ -96,4 +89,20 @@ async def receive_message(self, message_id: int) -> Mapping:
             if receive_message_id == message_id:
                 return response
 
-            self._responses_cache[receive_message_id] = response
+    @staticmethod
+    def _parse_runtime_evaluate(response: Mapping):
+        message = json.loads(response['params']['message'])
+        if message.get('error', None):
```

Comment:
`'error ' not in message` or `message.get('error', None) is not None`

Reply:
changed to `'error' in message`